lib/showcase.dart

```diff
@@ -55,10 +57,10 @@ class Showcase extends StatefulWidget {
                 : (disposeOnTap == null ? false : true),
             "disposeOnTap is required if you're using onTargetClick"),
         assert(
-        disposeOnTap == null
-            ? true
-            : (onTargetClick == null ? false : true),
-        "onTargetClick is required if you're using disposeOnTap"),
+            disposeOnTap == null
+                ? true
+                : (onTargetClick == null ? false : true),
+            "onTargetClick is required if you're using disposeOnTap"),
```

Comment:
Not sure why this is changed

Reply:
We have to use both disposeOnTap and onTargetClick together,
That's why we have added assert for disposeOnTap as well.

Reply:
Is it due to code format.

Reply:
Is it due to code format.